listenbrainz/config.py.sample

```diff
@@ -76,8 +76,8 @@ COUCHDB_USER = "listenbrainz"
 COUCHDB_ADMIN_KEY = "listenbrainz"
 
 # MusicBrainz OAuth
-MUSICBRAINZ_CLIENT_ID = "CLIENT_ID"
-MUSICBRAINZ_CLIENT_SECRET = "CLIENT_SECRET"
+MUSICBRAINZ_CLIENT_ID = "SO_FPZ4wKjl3NRIrIYE0EEqU7W3_UBo2"
```

Comment:
Ah, here we get to the point where "I'll just commit everything" becomes an issue :)

You always want to check exactly what you want to commit.

In this case, you modified the config *sample* file (you may also have modified the actual config.py file but changes to that one are ignored by git) and leaked your MB application credentials.

Not a biggie but we'll want to do two things:
1. revert this change (you can simply modify the file again to be "CLIENT_ID" and "CLIENT_SECRET", then commit that change)
2. you'll also want to delete your [MB application](https://musicbrainz.org/account/applications) and create a new one, that way the leaked credentials don't matter anymore